lib/src/main/java/dataframe/DataFrame.java

```diff
@@ -33,6 +33,10 @@ public List<Object[]> getRows() {
         return Arrays.stream(this.rows).map(Row::getValues).toList();
     }
 
+    public Schema getSchema(){
+        return this.schema;
```

Comment:
return the copy of schema. We do not want anyone to change the schema outside this dataframe